xonsh/prompt_toolkit_shell.py

```diff
@@ -46,6 +46,11 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.history = setup_history()
         self.pt_completer = PromptToolkitCompleter(self.completer, self.ctx)
+        self.mouse_support = builtins.__xonsh_env__.get('MOUSE_SUPPORT')
+        if builtins.__xonsh_env__.get('AUTO_SUGGEST') is True:
```

Comment:
This should just be `if builtins.__xonsh_env__.get('AUTO_SUGGEST'):`.  The `is True`, while probably safe here because of how `__xonsh_env__` works, should not generally be used.